website/pages/partials/jwt_claim_mapping_details.mdx

```diff
@@ -0,0 +1,71 @@
+## Trusted Identity Attributes via Claim Mappings
+
+Data from JWT claims can be returned from the authentication step as trusted
+identity attributes for use in binding rule selectors and bind name
+interpolation.
+
+Control of which claims are mapped to which identity attributes is governed by
+the [`ClaimMappings`](#claimmappings) and
+[`ListClaimMappings`](#listclaimmappings). These are both maps of items to copy
+with elements of the form: `"<JWT claim>":"<attribute suffix>"`.
+
+The only difference between these two types of mappings is that `ClaimMappings`
+is used to map values which can be interpolated while `ListClaimMappings` is
+used to map to map lists of values that cannot be interpolated.
+
+Assume this is your config snippet:
+
+```json
+{ ...other fields...
+  "ClaimMappings": {
+    "givenName": "first_name",
+    "surname": "last_name"
+  },
+  "ListClaimMappings": {
+    "groups": "groups"
+  }
+}
+```
+
+This specifies that the values in the JWT claims `"givenName"` and `"surname"`
+should be copied to attributes named `"value.first_name"` and
+`"value.last_name"` respectively. Additionally the list of values in the JWT
+claim `"groups"` should be copied to an attribute named `"list.groups"`.
+
+Concretely:
```

Comment:
Suggest something like. "The following table shows the resulting attributes that will be extracted, and the ways they may be used in Rule Bindings".